src/script/event/EventRepository.ts

```diff
@@ -565,14 +565,22 @@ export class EventRepository {
   private handleMessageUpdate(originalEvent: EventRecord, newEvent: MessageAddEvent) {
     const newEventData = newEvent.data;
     const originalData = originalEvent.data;
+
+    const isFailed = (status: StatusType | undefined): status is StatusType.FAILED => {
+      return status === StatusType.FAILED;
+    };
```

Comment:
was more thinking of this approach:


const isFailedEvent = (event: unknown): event is FailedEvent => event && event.status === StatusType.FAILED;

placed under the guards folder in the webapp